### ISSUE TITLE
Minecraft-Overviewer is not working-Ubuntu Desktop 18.04/ Minecraft-Overviewer funktioniert nicht (Ubuntu)

### ISSUE THREAD

**Issue by TheD3monEmper0r** [overviewer/Minecraft-Overviewer#1455]
(ENGLISH) Every time I try to start the Overviewer it gives me this Error:
command not found. Im using the Ubuntu Desktop version 18.04 !
Here is a screenshot:
![bildschirmfoto von 2018-05-16 17-58-45](https://user-images.githubusercontent.com/39342888/40129085-af64cd48-5933-11e8-82e2-458a30fbe065.png)
(GERMAN) Immer wenn ich versuche das den Overviewer zu starten kommt die Fehlermeldung das er den Befehl nicht kennt, ich weiß nicht woran das liegt da ich das Tutorial auf der Webseite befolgt habe und dann ja alles gehen müsste. Ich nutze die Ubuntu Desktop Version 18.04 !

**Comment by TheD3monEmper0r** [comment]
yeah It fixed it but now I have another problem that I cant fix:
![bildschirmfoto von 2018-05-16 20-29-32](https://user-images.githubusercontent.com/39342888/40136420-2cb520cc-5948-11e8-96bd-325e6a2cc520.png)
It says me I have the wrong path but the level.dat there, see my screenshot.

**Comment by CounterPillow** [comment]
That's a segmentation fault. Curious, I wonder why that happens. @agrif is this the Pillow ABI incompatibility again?

**Comment by SaintFenix** [comment]
As referenced in my own 18.04 ticket, I've downloaded this new version and can echo the issue that @TheD3monEmper0r is having.

**Comment by Capri205** [comment]
ok. downgrading pillow to 4.2.1 worked for me on Ubuntu, per the approach by ProfYaffle in issue 1423

https://github.com/overviewer/Minecraft-Overviewer/issues/1423

**Comment by SaintFenix** [comment]
Ran the command:

`sudo pip install 'Pillow<4.3.0'`

which forced Pillow to downgrade to 4.2.1 from 5.1.0 and overviewer is now running. :)

**Comment by treyzania** [comment]
This is also an issue on Debian Buster, but running the command above fixes the issue.

**Comment by temdur** [comment]
Had this issue. Downgrading pillow doesn't work.

But building from the sources does.

The downgraded pillow is still in my system. Maybe it is necessary for a proper build.

**Comment by CounterPillow** [comment]
Nope, Overviewer works with the latest Pillow. The problem is just our prebuilt packages.

**Comment by Taranchul** [comment]
I can confirm this on Kubuntu 18.04 64 bit. Building from source works, however. Since Ubuntu-based distributions are very widespread, and 18.04 being a Long Term Support release, may I kindly ask you to look into rebuilding the deb packages so that they don't segfault on those distros? (And hopefully, not break on other Debian-based distros in the process.)

Sorry, I first posted this in the closed duplicate #1449. I deleted it over there.
